CHEAT.md

```diff
@@ -0,0 +1,135 @@
+| Create a new stream                       | Set streamObject=##class(%Stream.GlobalCharacter).%New()<br>Set streamObject=##class(%Stream.GlobalBinary).%New()<br>Use methods below on a stream property |
+| Add text to a stream                      | Do streamObject.Write(text)<br>Do object.streamProperty.Write(text)                                                                                         |
+| Add a line of text to a stream            | Do streamObject.WriteLine(text)<br>Do object.streamProperty.WriteLine(text)                                                                                 |
+| Read len characters of text from a stream | Write streamObject.Read(len)<br>Write object.streamProperty.Read(len)                                                                                       |
+| Read a line of text from a stream         | Write streamObject.ReadLine(len)<br>Write object.streamProperty.ReadLine(len)                                                                               |
+| Go to the beginning of a stream           | Do streamObject.Rewind()<br>Do object.streamProperty.Rewind()                                                                                               |
+| Go to the end of a stream, for appending  | Do streamObject.MoveToEnd()<br>Do object.streamProperty.MoveToEnd()                                                                                         |
+| Clear a stream                            | Do streamObject.Clear()<br>Do object.streamProperty.Clear()                                                                                                 |
+| Display the length of a stream            | Write streamObject.Size<br>Write object.streamProperty.Size                                                                                                 |
+
+## Unit Testing Macros
+
+| Action                      | Code                                           |
+|-----------------------------|------------------------------------------------|
+| Assert equality             | Do $$$AssertEquals(value1, value2, message)    |
+| Assert inequality           | Do $$$AssertNotEquals(value1, value2, message) |
+| Assert status is OK         | Do $$$AssertStatusOK(status, message)          |
+| Assert status isn't OK      | Do $$$AssertStatusNotOK(status, message)       |
+| Assert condition is true    | Do $$$AssertTrue(condition, message)           |
+| Assert condition isn't true | Do $$$AssertNotTrue(condition, message)        |
+| Log message                 | Do $$$LogMessage(message)                      |
+
+## Other Macros
+
+| Action                           | Code                                    |
+|----------------------------------|-----------------------------------------|
+| Return a good status             | Quit $$$OK                              |
+| Return an error status           | Quit $$$ERROR($$$GeneralError, message) |
```

Comment:
Added in https://github.com/intersystems-ru/cos-guidelines/commit/7e42b52bd7190b187a0aebbda97f7c27f33736ca